templates/ffc-template-node/template.yaml

```diff
@@ -0,0 +1,158 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: ffc-template-node
+  title: "FFC Template Node"
+  description: "Template to support rapid delivery of microservices for FFC Platform. It contains the configuration needed to deploy a simple Hapi Node server to the Azure Kubernetes Platform."
+  tags:
+    - node
+    - hapi
+    - aks
+    - fcp
+spec:
+  owner: fcp
+  type: service
+
+  parameters:
+    - title: Component Information
+      description: Enter properties describing this component.
+      required:
+        - component_id
+        - description
+        - owner
+        - component_type
+        - component_lifecycle
+      properties:
+        component_id:
+          title: Component Name
+          type: string
+          description: Unique name of the component which will be used for the repository, package, and image names. This should be in the format <program>-<workstream>-<repo>. Maximum 64 characters.
+          ui:field: EntityNamePicker
+          ui:autofocus: true
+
+        description:
+          title: Description
+          type: string
+          description: Describe what this component does
+
+        system:
+          title: System
+          type: string
+          description: Select the system that this component will be a part of
+          ui:field: EntityPicker
+          ui:options:
+            allowArbitraryValues: false
+            catalogFilter:
+              - kind: System
+
+        component_type:
+          title: Component Type
+          type: string
+          description: Select the type of this component. Allowed values - Service (a backend service or API), Web (a web application), Library (a library or package that can be consumed by other applications)
+          default: service
+          enum:
+            - service
```

Comment:
Can we discuss the allowed component types
Frontend service, Backend service

How do we differentiate APIs vs Web UI services?